typing/jkind.ml

```diff
@@ -2042,6 +2373,12 @@ module Format_history = struct
     | Wildcard -> fprintf ppf "it's a _ in the type"
     | Unification_var -> fprintf ppf "it's a fresh unification variable"
 
+  let format_abstract_creation_reason ppf :
+      History.abstract_creation_reason -> unit = function
+    | Strengthening (id, p) ->
```

Comment:
This creation reason is never actually used anywhere - nowhere in the codebase constructs this variant.